modules/data.land/R/ic_process.R

```diff
@@ -12,12 +12,16 @@ ic_process <- function(settings, input, dir, overwrite = FALSE){
   #--------------------------------------------------------------------------------------------------#
   # Extract info from settings and setup
   site       <- settings$run$site
-  start_date <- settings$run$start.date
-  end_date   <- settings$run$end.date
+  run_start  <- settings$run$start.date
   model      <- settings$model$type
   host       <- settings$host
   dbparms    <- settings$database
   
+  # Handle IC Workflow locally
+  if(host$name != "localhost"){
+    host$name <- "localhost"
```

Comment:
for (1) I don't overwrite `settings$host`, and use it in the end to sync the files.. for (2) all the local directories are derived from `dir` string which is reassigned to the local directories in the next line